ui/src/stylesheets/specific/navigation.scss

```diff
@@ -57,7 +57,6 @@
   list-style: none;
   line-height: 1.7em;
   max-height: 55vh;
```

Comment:
I think this should also be moved where `overflow-y` was moved. There doesn't seem to be any practical consequences to having this here, but I really think it should be applied specifically to the top-level `.nav-list`, just in case.